src/backend/api/Fusion.Resources.Domain/Queries/GetResourceOwnerProfile.cs

```diff
@@ -50,9 +49,9 @@ public async Task<QueryResourceOwnerProfile> Handle(GetResourceOwnerProfile requ
                 var user = await profileResolver.ResolvePersonBasicProfileAsync(request.ProfileId.OriginalIdentifier);
 
                 if (user is null)
-                    throw new InvalidOperationException($"Could not resolve profile '{request.ProfileId.OriginalIdentifier}'");
+                    throw new PersonNotFoundError(request.ProfileId.OriginalIdentifier);
                 if (user.FullDepartment is null) 
```

Comment:
As far as I know, all users have a department. (At least from LineOrg)
And wondering about=> FullDepartment null => not authorized ?